fuller/mrfRec.py

```diff
@@ -258,13 +270,15 @@ def normalizeI(
         try:
             from mclahe import mclahe
 
-            self.I = mclahe(
-                self.I,
-                kernel_size=kernel_size,
-                n_bins=n_bins,
-                clip_limit=clip_limit,
-                use_gpu=use_gpu,
-            )
+            for t in range(self.lengthdelay):
+                self.I[:,:,:,t] = mclahe(
+                    self.I[:,:,:,t] ,
+                    kernel_size=kernel_size,
+                    n_bins=n_bins,
+                    clip_limit=clip_limit,
+                    use_gpu=use_gpu,
+                )
```

Comment:
we should discuss if this makes sense. Maybe generate one set of parameters, and apply everywhere. Not sure if mclahe can do this rn.